core/src/main/java/com/wepay/kafka/connect/bigquery/config/BigQuerySinkConfig.java

```diff
@@ -69,20 +75,21 @@
       "Names for the datasets kafka topics will write to "
       + "(form of <topic regex>=<dataset>)";
 
+  public static final String SCHEMA_RETRIEVER_CONFIG =         "schemaRetriever";
+  private static final ConfigDef.Type SCHEMA_RETRIEVER_TYPE =  ConfigDef.Type.CLASS;
+  private static final Class<?> SCHEMA_RETRIEVER_DEFAULT =     null;
```

Comment:
Thinking of just adding another Gradle task (something like `confluentTarBall`) in the kcbq-connector directory that just combines the connector and confluent tarballs.